tests/testthat/test-warnings.R

```diff
@@ -43,3 +43,19 @@ test_that("Intercept exceptions",
         },"Y")
     })
 })
+
+test_that("Checking for multiple data sets in the environment", {
+    
+    expect_warning(WarnIfVariablesSelectedFromMultipleDataSets(), NA)
```

Comment:
This has the potential to fail and cause a dev to be confused if they have stuff in their global env. Maybe add a check and skip the tests if there is stuff in the global env?